report/3.Theory/LanguageSpecification.tex

```diff
@@ -126,6 +132,29 @@ \section{Language Specification}
 Not having operators for arithmetic operations also means that there are no native operator precedence in the language.
 This can be amended by structuring calculations in a left-to-right manner, or by encapsulating operations with parenthesis. 
 
+\textbf{High order functions}\\
+As mentioned in section \ref{subsec:hofhcl} the language will include high-order functions. 
+To declare that a function will accept other functions the developer should simply declare a parameter of type "func" with the expected input and output types from the function.
```

Comment:
comma after functions
"..accept other functions, that the.."

editor note: i know i rant about too many commas.. but sometimes commas do need a bit of love